docs/concepts/linq.rst

```diff
@@ -0,0 +1,268 @@
+LINQ
+====
+
+What is it?
+-----------
+
+LINQ provides language-level querying capabilities and `higher-order function <https://en.wikipedia.org/wiki/Higher-order_function>`_ API to C# and VB (note that F# already has this capability by nature of being a functional programming language).  It is used either via native query syntax,
```

Comment:
Hm, weren't higher-order functions already partially supported via delegates, and especially anonyous delegates?